cli/server/previewServer.js

```diff
@@ -30,6 +30,26 @@ const PUBLIC_CONFIG_BASENAME = 'cache-public.config';
 const PUBLIC_CONFIG_PATH = path.join(CACHE_DIRECTORY, PUBLIC_CONFIG_BASENAME);
 const FAV_ICON_PATH = './node_modules/jest-preview/cli/server/favicon.ico';
 
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk'); // Already installed in `update-notifier`
+
+// Checks for available update
+const notifier = updateNotifier({
+  pkg: require('../../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
+  shouldNotifyInNpmScript: true, // Allows notification to be shown when running as an npm script
+  distTag: 'latest' // Can be use to notify user about pre-relase version
```

Comment:
@huynhducduy  I want to discuss the testing strategy for this feature. Should I change this to `alpha`, then try to release a few versions with `alpha` tags. Then keep running the old version of `jest-preview` to see if a notifier shows up?

Reply:
@nvh95 You dont need to release anything, just take another package that already have `alpha` things and use the [pkg](https://github.com/yeoman/update-notifier#pkg) option.

Reply:
That's clever!